static/app/utils/events.tsx

```diff
@@ -63,7 +50,12 @@ function computeTitleWithTreeLabel(title: string | undefined, metadata: EventMet
 function computeTitleWithTreeLabel(metadata: EventMetadata) {
   const {type, current_tree_label, finest_tree_label} = metadata;
   const treeLabel = current_tree_label || finest_tree_label;
-  const formattedTreeLabel = treeLabel ? treeLabel.join(' | ') : undefined;
+  const formattedTreeLabel = treeLabel
+    ? treeLabel
+        .map(part => part?.function || part?.package)
```

Comment:
Would it make sense to add `|| part?.type` here, like in eventTitleTreeLabel.tsx?

Reply:
Right now `part.type` is not emitted by the backend so it shouldn't matter, but I added it.

Reply:
in case we would like to have a structure similar to the types, then we will need to update this logic